### PR TITLE
Polishing (EXPOSUREAPP-6913)

### PR DESCRIPTION
**Pull request by mtwalli**
- Finalise UI changes such screen title 
- Navigation to Vaccination list
- Connect it in the actual graph (not accessible atm)
- Introduced deeplinking for vaccination list to be able to navigate deeply in vaccination graph

### CODE REVIEW

**Review by LukasLechnerDev — DISMISSED** [review]
LGTM. Optional renaming.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentEvents.kt [hunk 1]
```diff
@@ -8,7 +8,9 @@ sealed class HomeFragmentEvents {
 
     object ShowErrorResetDialog : HomeFragmentEvents()
 
+    object GoToStatisticsExplanation : HomeFragmentEvents()
+
     data class ShowDeleteTestDialog(val type: CoronaTest.Type) : HomeFragmentEvents()
 
-    object GoToStatisticsExplanation : HomeFragmentEvents()
+    data class VaccinationList(val vaccinatedPersonIdentifier: String) : HomeFragmentEvents()
```
[the remark this answers is not in the document]
Comment (mtwalli) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentEvents.kt:15]: make sense

**Review by d4rken — APPROVED** [review]
lgtm

**Review by LukasLechnerDev — APPROVED** [review]

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Pull request merged**